workloads/README.md

```diff
@@ -0,0 +1,56 @@
+## Deploying AgnosticD workloads
+
+This is a collection of scripts that help you deploy workloads on AgnosticD Openshift clusters. It includes sample configuration files for migration related workloads.
```

Comment:
Openshift -> OpenShift

Reply:
I feel we could more clearly represent what this directory contains at the beginning of this README.

E.g.
```
It includes sample configuration files for migration related workloads.
```
Should perhaps become...
```
These AgnosticD workloads assist in automated deployment of:
 - The OpenShift 3->4 Application Migration Tool 
 - Various sample apps for use when testing App Migration capabilities from OpenShift 3->4